custom_components/unifi_insights/api/protect/models/doorlock.py

```
@@ -0,0 +1,32 @@
+"""Door lock models for UniFi Protect API.
+
+The 7.1.87 integration REST API does not expose a dedicated ``/doorlocks``
+resource; door locks are surfaced through the ``doorlock`` WebSocket
+``modelKey`` (see ``WS_DEVICE_UPDATE_TYPES``). This tolerant model is used to
+parse those WebSocket payloads.
+"""
```

Ruff (0.16.0)

[warning] Multi-line docstring summary should start at the second line

Insert line break and indentation after opening quotes

(D213)